apps/android/package.json

```diff
@@ -22,6 +22,7 @@
   },
   "dependencies": {
     "@fluentui-react-native/tester": "^0.27.1",
+    "@react-native-community/slider": "^3.0.3",
```

Comment:
Can you add this to the iOS package.json too?

Reply:
Oh is it required? Like the slider is functional on iOS already. Heres a quick demo!

https://user-images.githubusercontent.com/23552252/123345697-437b2a80-d574-11eb-8a10-7a8c90ca394e.mov

Reply:
Not required, but good for pedantic-ness =)
Our test apps are structured weirdly, and missing dips in package.json have caused weird errors in the past. Also I want to combine the test apps one day so this would be good to know the actual set per app

Reply:
Can you add that video to the PR description? Thanks!

Reply:
Also the desktop screenshots. Maybe just have one screenshot per platform instead of showing off each individual prop, for ease of looking back at this PR in the future =)